entrypoint.sh

```diff
@@ -40,3 +40,33 @@ echo "Uploading wheels to anaconda.org..."
 
 anaconda -t $ANACONDA_TOKEN upload --force -u "$ANACONDA_ORG" "$INPUT_ARTIFACTS_PATH"/*.whl
 echo "Index: https://pypi.anaconda.org/$ANACONDA_ORG/simple"
+
+# Get the package name from the "${INPUT_ARTIFACTS_PATH}"/*.whl files
+if [ -d "${INPUT_ARTIFACTS_PATH}" ]; then
+    # Leave off *.whl in ls target to avoid directory name prefix on wheel_name
+    wheel_name=$(ls --almost-all "${INPUT_ARTIFACTS_PATH}" | head --lines 1)
```

Comment:
I'm not 100% sure this is POSIX and not just Linux, but it works to not get the `dist/` prefix here

```console
$ ls --almost-all dist/ | head --lines 1
xarray-2023.5.1.dev10+gf45eb733-py3-none-any.whl
```